qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -109,3 +109,7 @@
                     cpu_model_flags += ",mce=off"
                     flags = ""
                     no_flags = "mce"
+                - test_0x40000001:
+                    only Linux
+                    start_vm = yes
```

Comment:
Maybe not necessary?

Reply:
There has defined "start_vm = no" in Line 5.